---
title: "Microbiomics, metabolomics, predicted metagenomics and hepatic steatosis in a population‐based study of 1355 adults"
authors:
- Louise JM Alferink
- Djawad Radjabzadeh
- Nicole S Erler
- Dina Vojinovic
- Carolina Medina‐Gomez
- André G Uitterlinden
- Robert J de Knegt
- Najaf Amin
- M Arfan Ikram
- Harry LA Janssen
- Jessica C Kiefte-deJong
- Herold J Metselaar
- Cornelia M van Duijn
- Robert Kraaij
- Sarwa Darwish Murad

date: "2020-06-12"
doi: "10.1002/hep.31417"

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Hepatology*, 2020"
publication_short: ""

abstract: "#### Background & Aims\n
Previous small studies have appraised the gut microbiome (GM) in steatosis, but
large‐scale studies are lacking. We studied the association of GM diversity and
composition, plasma metabolites, predicted functional metagenomics and
steatosis.

##### Approach & Results\n
This is a cross‐sectional analysis of the prospective population‐based Rotterdam
Study. We used 16Sribosomal‐RNA gene sequencing and determined taxonomy using
the Silva‐reference database. Alpha‐and beta‐diversity were calculated using
Shannon‐index and Bray‐Curtis dissimilarities. Differences were tested across
steatosis using PerMANOVA. Hepatic steatosis was diagnosed by ultrasonography.
We subsequently selected genera using regularized regression. The functional
metagenome was predicted based on the GM using KEGG‐pathways. Serum metabolomics
were assessed using high‐throughput proton nuclear magnetic resonance. All
analyses were adjusted for age, sex, BMI, alcohol, diet, and proton‐pump
inhibitors. We included 1355 participants of which 472 had steatosis.
Alpha‐diversity was lower in steatosis (P=1.1&times;10<sup>9</sup>) and beta‐diversity varied
across steatosis strata (P=0.001). Lasso selected 37 genera of which three
remained significantly associated after adjustment (*Coprococcus3*: β=‐65;
*Ruminococcus Gauvreauiigroup*: β=62; and *Ruminococcus Gnavusgroup*: β=45, Q‐value
=0.037). Predicted metagenome analyses revealed that pathways of secondary
bile‐acid synthesis and biotin metabolism were present and D‐alanine metabolism
was absent in steatosis. Metabolic profiles showed positive associations for
aromatic‐and branched chain amino acids and glycoprotein acetyls with steatosis
and *R. Gnavusgroup*, whereas these metabolites were inversely associated with
alpha‐diversity and *Coprococcus3*.


#### Conclusions:\n
We confirmed, for the first time on a large‐scale, the lower microbial diversity
and association of *Coprococcus* and *Ruminococcus Gnavus* with steatosis. We
additionally showed that steatosis and alpha-diversity share -opposite-
metabolic profiles."

# Summary. An optional shortened abstract.
summary: ""

tags:
- gut microbiome
- hepatic steatosis
- metabolomics
- metagenomics
- liver stiffness

featured: false

links:
 - name: "Supplementary material"
   url: "https://aasldpubs.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fhep.31417&file=hep31417-sup-0001-Suppinfo.pdf"
url_pdf: 'https://aasldpubs.onlinelibrary.wiley.com/doi/epdf/10.1002/hep.31417'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
focal_point: ""
preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: example
---
